_posts/2019-10-10-bazel-1.0.md

```diff
@@ -0,0 +1,99 @@
+*   [`--incompatible_auto_configure_host_platform`](https://github.com/bazelbuild/bazel/issues/7081)
+*   [`--incompatible_disable_legacy_proto_provider`](https://github.com/bazelbuild/bazel/issues/7152)
+*   [`--incompatible_disable_nocopts`](https://github.com/bazelbuild/bazel/issues/8706)
+*   [`--incompatible_disable_partition_default_parameter`](https://github.com/bazelbuild/bazel/issues/8953)
+*   [`--incompatible_disable_proto_source_root`](https://github.com/bazelbuild/bazel/issues/7153)
+*   [`--incompatible_disable_static_cc_toolchains`](https://github.com/bazelbuild/bazel/issues/8546)
+*   [`--incompatible_disallow_hashing_frozen_mutables`](https://github.com/bazelbuild/bazel/issues/7800)
+*   [`--incompatible_disallow_legacy_java_provider`](https://github.com/bazelbuild/bazel/issues/7598)
+*   [`--incompatible_disallow_legacy_javainfo`](https://github.com/bazelbuild/bazel/issues/5821)
+*   [`--incompatible_disallow_rule_execution_platform_constraints_allowed`](https://github.com/bazelbuild/bazel/issues/8136)
+*   [`--incompatible_disallow_split_empty_separator`](https://github.com/bazelbuild/bazel/issues/7355)
+*   [`--incompatible_do_not_emit_buggy_external_repo_import`](https://github.com/bazelbuild/bazel/issues/8711)
+*   [`--incompatible_enable_profile_by_default`](https://github.com/bazelbuild/bazel/issues/8783)
+*   [`--incompatible_generated_protos_in_virtual_imports`](https://github.com/bazelbuild/bazel/issues/9215)
+*   [`--incompatible_make_thinlto_command_lines_standalone`](https://github.com/bazelbuild/bazel/issues/6791)
+*   [`--incompatible_no_target_output_group`](https://github.com/bazelbuild/bazel/issues/7949)
+*   [`--incompatible_remove_legacy_whole_archive`](https://github.com/bazelbuild/bazel/issues/7362)
+*   [`--incompatible_require_java_toolchain_header_compiler_direct`](https://github.com/bazelbuild/bazel/issues/7990)
+*   [`--incompatible_restrict_attribute_names`](https://github.com/bazelbuild/bazel/issues/6437)
+*   [`--incompatible_restrict_named_params`](https://github.com/bazelbuild/bazel/issues/8147)
+*   [`--incompatible_skip_genfiles_symlink`](https://github.com/bazelbuild/bazel/issues/8651)
+*   [`--incompatible_tls_enabled_removed`](https://github.com/bazelbuild/bazel/issues/8061)
+*   [`--incompatible_use_aapt2_by_default`](https://github.com/bazelbuild/bazel/issues/6907)
+*   [`--incompatible_use_native_patch`](https://github.com/bazelbuild/bazel/issues/8316)
+*   [`--incompatible_use_specific_tool_files`](https://github.com/bazelbuild/bazel/issues/8531)
+*   [`--incompatible_windows_bashless_run_command`](https://github.com/bazelbuild/bazel/issues/8240)
+*   [`--incompatible_windows_native_test_wrapper`](https://github.com/bazelbuild/bazel/issues/6622)`
+
+
+## Windows
+
+
+
+*   Genrule now supports `cmd_bash`, `cmd_ps`, `cmd_bat` attributes. More details in [`genrule` documentation](https://docs.bazel.build/versions/master/be/general.html#genrule.cmd)
```

Comment:
My suggestion was not applied?

Reply:
That was my suggested rephrase:

```
* [`genrule`](https://docs.bazel.build/versions/master/be/general.html#genrule) now supports `cmd_bash`, `cmd_ps`, and `cmd_bat` attributes for better integration on Windows.
```

Reply:
Ah now I see: I missed the rest of your suggestion beyond the right scroll